spec/support/vcr_cassettes/sm_client/messages/nested_resources/gets_a_single_attachment_by_id.yml

```diff
@@ -24,7 +24,7 @@ http_interactions:
       Server:
       - Apache-Coyote/1.1
       Content-Disposition:
-      - attachment; filename=noise300x200.png
+      - attachment; filename=%22noise300x200.png%22
```

Comment:
Wasn't sure whether to set this as `filename=%22noise300x200.png%22` or `filename="noise300x200.png"`, but the replace code I added covers both cases.